### DISCUSSION TITLE
Unable to open link to PR

### DISCUSSION THREAD

**Discussion by binury** [community/community#68156] [Mobile] [answered]
<details>
<summary>
Opening the following link https://github.com/estools/esquery/pull/134 will fail in the GH mobile app both when tapping a link organically and when using the in-app paste-to-go feature. This doesn't occur with other PRs.

![image](https://github.com/community/community/assets/1146921/89f97509-113d-4f83-86a3-47c89b13ff52)
</summary>
GitHub for iOS, Version 1.131.0 (110081196)
Device: iPhone 13 Pro Max (iOS 17.0)
</details>

**Comment by alcere** [comment]
Hi @binury! We are sorry for the troubles! I can reproduce the same issue, I will open an internal issue to fix this bug. Thanks for reporting!

**Comment by mattruggio** [comment, marked as answer]
@binury - Thanks for the heads up on this. This looks to functioning as expected now, could you possibly confirm this as well?

**Reply by binury** [reply]
@mattruggio Looks like the issue is resolved 🙂 Curious- what was it about certain PRs that made them unable to be loaded in the mobile app?
